examples/07_automatic_preprocessing_with_the_supervectorizer.py

```diff
@@ -0,0 +1,126 @@
+"""
+Automatic pre-processing with the SuperVectorizer
+=================================================
+
+In this notebook, we will illustrate the use of the `SuperVectorizer`, which
+can automatically classify columns of a dataset based on their type, and apply
+transformers based on this classification.
+To demonstrate that, we will use the `employee salaries` dataset.
+"""
+
+###############################################################################
+# Import the data
+# ---------------
+#
+# Let's fetch the dataset, and load X and y:
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from dirty_cat.datasets import fetch_employee_salaries
+
+employee_salaries = fetch_employee_salaries()
+print(employee_salaries['DESCR'])
+
+X: pd.DataFrame = employee_salaries['data']
+y = employee_salaries['target']
+# We'll drop a few columns we don't want
+X.drop(
+    [
+        'Current Annual Salary',  # Too linked with target
+        'full_name',  # Not relevant to the analysis
+        '2016_gross_pay_received',  # Too linked with target
+        '2016_overtime_pay',  # Too linked with target
+        'date_first_hired'  # Redundant with "year_first_hired"
+    ],
+    axis=1,
+    inplace=True
+)
+
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.15, random_state=42)
```

Comment:
```suggestion
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.15,
                                                                            random_state=42)
```
I think that this line is a bit long, no?